### ISSUE TITLE
Unable to download "openvino-models.zip" - Error "File wasn't available on site"

### ISSUE THREAD

**Issue by abirch293** [intel/openvino-plugins-ai-audacity#116]
![image](https://github.com/intel/openvino-plugins-ai-audacity/assets/60498128/1b9fc3a4-0422-454f-be52-ab86e7d10d8d)

**Comment by RyanMetcalfeInt8** [comment]
Hi @abirch293,

Hmm, it still seems to be downloading for me. Do you have another PC you can try from?

Thanks,
Ryan

**Comment by abirch293** [comment]
Hey Ryan!

Thanks for the quick response. I already had this file on another computer that I transferred over after I saw the error. I was able to complete the download again on the spare PC (just to be sure it was all me), and it succeeded. Must have been bad timing on my part! The download didn't show the error until it was complete, as odd as that sounds. I submitted the just in case others were having issues as well. It appears to be either a blip or this computer. 

Thanks again! I'm all setup :) 
Cheers!
